### PR TITLE
UofT-DSI | Applying Statistical Concepts - Assignment 2

### PR DESCRIPTION
**Pull request by asnjunior**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Submitting assignment 2

## What did you learn from the changes you have made?
How to linear regression in python

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
NA

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
NA

## How were these changes tested?
By splitting the data between training and test set 75% - 25%.

## A reference to a related issue in your repository (if applicable)
NA

## Checklist
- [ x ] I can confirm that my changes are working as intended

### CODE REVIEW

**Comment by KasraVakiloroayaei** [comment]
A2: INCOMPLETE 
Please fix the following questions in order to complete A2:

Q2 part (iv): verify the number of predictors in the model
Q3: The coefficients and intercept for each predictor is not correct. You may have forgotten to add model_year to your linear fit model 
Q4: Please take a look at your code to ensure you are getting the correct test set RMSE

**Comment by asnjunior** [comment]
Hi, Melokazz. Thanks for the feedback. I assume instead of  Q2 part (iv) you meant Q1 part (iv). I have updated my answer to that question. I have also added model_year to my predictors so my coefficients are updated now. For Q4, in the assignment it is asking for RMSPE thus I used the test set. I noticed though you mentioned RMSE in which case the training set should be used and maybe this is where the confusion is? But since I also missed model_year as a predictor in this part, i get an updated value of RMSPE.

**Comment by KasraVakiloroayaei** [comment]
A2: Complete 

great work making the changes noted above! I am happy your were able to find a solution
